src/ore/src/tracing.rs

```diff
@@ -141,11 +141,11 @@ pub struct TracingConfig<'a> {
 
 /// Configures tracing according to the provided command-line arguments.
 /// Returns a `Write` stream that represents the main place `tracing` will
-/// log to.
+/// log to, and a `tracing::Level` that is the `librdkafka` debug level.
```

Comment:
To keep abstraction boundaries clean, `ore` should not know anything about `librdkafka`. Is there something else we can return here instead? The filter maybe?

Reply:
We could take in a list of `targets` and return a list of `Level`s for those targets we checked?